server-web/actix-rest/openapi.yaml

```diff
@@ -0,0 +1,51 @@
+# An example OpenAPI spec
```

Comment:
Where does this file get used? Is something being autogenerated?

This also seems like it could be in common-rest instead of actix-rest

Reply:
This is manually generated (but can also be generated programmatically. Just putting it out there as an example of what OpenAPI protocol looks like). Moved to common-rest.